src/components/VideoCard/VideoCard.jsx

```diff
@@ -1,15 +1,23 @@
+import { useState } from 'react';
 import { thumbnailLink } from '../../utils/video-details-services';
+import { OverflowMenu } from '../OverflowMenu/OverflowMenu';
 
 import './VideoCard.css';
 
 const VideoCard = ({ video }) => {
-
     const reduceTitleLength = (title) => {
-        if(title.length > 50)
-            return title.substring(0, 47)+'...';
+        if (title.length > 50) return title.substring(0, 47) + '...';
```

Comment:
Suggestion: You can use ellipsis to trim the title to a specific number of lines.

Reply:
Will make the change. Thanks!